relocalization:
    surface_icp_corr_threshold: 0.6
    surface_icp_max_iternum: 50
    surface_icp_fit_threshold: 0.05 
    surface_pass_filter_threshold: 110
    corner_icp_corr_threshold: 0.6
    corner_icp_max_iternum: 50
    corner_icp_fit_threshold: 0.05 
    corner_pass_filter_threshold: 100
    key_frame_num: 26
    accumulate_num: 1
    keypoints_num_threshold: 100
    fit_score_threshold: 0.1
    corner_initial_change_scope: 0.3
    surface_initial_change_scope: 0.3
